src/clusterfuzz/_internal/common/testcase_utils.py

```diff
@@ -61,15 +60,30 @@ def emit_testcase_triage_duration_metric(testcase_id: int, step: str):
                  ' failed to emit TESTCASE_UPLOAD_TRIAGE_DURATION metric.')
     return
 
+  from_fuzzer = not get_testcase_upload_metadata(testcase_id)
+
+  assert step in [
+      'analyze_launched', 'analyze_completed', 'minimize_completed',
+      'regression_completed', 'impact_completed', 'issue_updated'
+  ]
+
+  if not testcase.get_age_in_seconds():
+    logs.warning(f'No timestamp associated to testcase {testcase_id},'
+                 ' failed to emit TESTCASE_UPLOAD_TRIAGE_DURATION metric.')
+    return
+
+  testcase_age_in_hours = testcase.get_age_in_seconds() / 3600
+
   logs.info('Emiting TESTCASE_UPLOAD_TRIAGE_DURATION metric for testcase '
-            f'{testcase_id} (age = {elapsed_time_since_upload}) '
+            f'{testcase_id} (age = {testcase_age_in_hours} hours.) '
             'in step {step}.')
 
   monitoring_metrics.TESTCASE_UPLOAD_TRIAGE_DURATION.add(
```

Comment:
The naming of this metric is now inaccurate since it now covers both fuzzed and uploaded testcases, it needs to be changed. 
Does `BUG_FILING_FROM_TESTCASE_ELAPSED_TIME` cover both also?